website/docs/components/button/partials/code/how-to-use.md

```diff
@@ -132,5 +132,5 @@ If using an `@href` or `@route` and needing to disable the component, you’ll n
 !!!
 
 ```handlebars
-<Hds::Button @text="Copy to clipboard" disabled />
+<Hds::Button @text="Alert me" disabled />
```

Comment:
probably I would go the whole hog and add an event handler also here to show that it doesn't fire on a disable button:
```suggestion
<Hds::Button @text="Alert me" disabled {{on "click" this.alertOnClick}} />
```